package-lock.json

```diff
@@ -9,15 +9,15 @@
       "resolved": "https://registry.npmjs.org/@kubernetes/typescript-node/-/typescript-node-0.1.1.tgz",
       "integrity": "sha1-6uyJ10fT4vIunYS2V/0u0UP5F5s=",
```

Comment:
revert. pacakge-lock changed without changes to package.json.

Reply:
reverted the file.

Reply:
Revert changes to this file

Reply:
revert changes to this file

Reply:
done